utils/src/main/kotlin/Constants.kt

```diff
@@ -45,6 +45,11 @@ const val ORT_DATA_DIR_ENV_NAME = "ORT_DATA_DIR"
  */
 const val ORT_CONFIG_FILENAME = "ort.conf"
 
+/**
+ * The name of the ORT package configurations directory.
+ */
+const val ORT_PACKAGE_CONFIGURATIONS_DIRNAME = "package-configurations"
```

Comment:
Consider "packages" instead.

Reply:
I think I'd prefer to stick with a name derived from the plural name of the underlying `PackageConfiguration` class, and just have an "s" after "Configurations".